core-bundle/src/Resources/contao/templates/analytics/analytics_google.html5

```diff
@@ -1,16 +1,17 @@
 <?php
 
 // To use this script, please fill in your Google Analytics ID below
-$GoogleAnalyticsId = 'UA-XXXXX-X';
+$GoogleAnalyticsId = 'G-XXXXXXXXXX';
 
 // DO NOT EDIT ANYTHING BELOW THIS LINE UNLESS YOU KNOW WHAT YOU ARE DOING!
-if ('UA-XXXXX-X' != $GoogleAnalyticsId && !BE_USER_LOGGED_IN && !$this->hasAuthenticatedBackendUser()): ?>
+if ('G-XXXXXXXXXX' != $GoogleAnalyticsId && !BE_USER_LOGGED_IN && !$this->hasAuthenticatedBackendUser()): ?>
 
+<script async src="https://www.googletagmanager.com/gtag/js?id=<?= $GoogleAnalyticsId ?>"></script>
 <script>
-  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)})(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
-  ga('create', '<?= $GoogleAnalyticsId ?>', 'auto');
-  ga('set', 'anonymizeIp', true);
-  ga('send', 'pageview');
+  window.dataLayer = window.dataLayer || [];
+  function gtag(){dataLayer.push(arguments);}
+  gtag('js', new Date());
+  gtag('config', '<?= $GoogleAnalyticsId ?>');
```

Comment:
This is a default now:

https://support.google.com/analytics/answer/2763052?hl=en

> In Google Analytics 4, IP anonymization is not necessary since IP addresses are not logged or stored.

https://support.google.com/analytics/answer/9310895?hl=en&ref_topic=11192706#gtagjs-configuration

> IP Anonymization is standardized to "true" and not configurable in a Google Analytics 4 property. As such, the IP address is automatically masked with the standard event to Google Analytics 4 property ("G-XXXXXXXX").

Reply:
That only applies to GA4 properties. If someone still uses `UA-XXXXXXXX`, they will have to request IP anonymization by adding `{ 'anonymize_ip': true }`, won‘t they?

Reply:
If someone already uses UA, they probably already have the customized template and will not be affected by this update? For new sites, GA4 is default choice, and this template provides a default configuration for that.

That being said, there is no harm in keeping a deprecated setting either. Should I still add it explicitly?

NOTE: GA4 will be the only choice for new properties after July 1st, 2023.

Reply:
@contao/developers WDYT?

Reply:
I wonder if Contao 5 should ship this template at all. Why does it provide defaults for providers such as Google but not privacy focussed ones like plausible.io? We provide a super easy way to include whatever scripts you want to - imho that's all we need to do.

Reply:
I didn't realize this is a PR against 4.9, sorry. That being said, I don't have any preference at all. Again, I don't see why we would ship those templates and I haven't used any Google Analytics services for years.

Reply:
I agree with @Toflar for Contao 5 not to ship any of the templates for analytics. Every tracking software provides the tracking code and with a simple copy & paste you can create your own tracking snippet and you already have to improve the existing templates.

For Contao 4.x I would update to GA4 because as @ameotoko said, there's no possibility creating a new GA3 property. We have to make a comment in the template that it's GA4.